txp/Pos/Txp/Toil/Failure.hs

```diff
@@ -28,8 +37,23 @@ data ToilVerFailure
                   , tOutputSum :: !Integer}
     | ToilInconsistentTxAux !Text
     | ToilInvalidOutputs !Text  -- [CSL-814] TODO: make it more informative
```

Comment:
@neongreen now can you please create a new issue for this TODO and change the reference in the code (I don't like TODO with a reference to already closed issue)?